src/ingest_validation_tools/enums.py

```diff
@@ -87,3 +67,27 @@
         'phosphopeptides'
     ]
 }
+
+def _test_assay_name_compatibility():
+    """
+    >>> import requests
+    >>> local_names = shared_enums['assay_type']
+    >>> remote_mismatch = []
+    >>> for name in local_names:
+    ...     response = requests.post(
+    ...         'https://search.api.hubmapconsortium.org/assayname',
+    ...         json={'name':name}, headers={'Content-Type': 'application/json'}).json()
+    ...     if 'error' in response:
+    ...         remote_mismatch.append(name)
+    >>> print('\\n'.join(remote_mismatch))
+    <BLANKLINE>
+    """
+    # The list above should be empty: That means that all the assays
+    # listed below are recognized by the assay service.
+    # TODO: Joel will progressively uncomment names below, and when that is done
+    #       _validate_level_1_enum() can be renabled in schema_loader.py.
+    #       https://github.com/hubmapconsortium/ingest-validation-tools/issues/1023
+    pass
+
+if __name__ == "__main__":
+    main()
```

Comment:
There is no `main` defined.

Reply:
Ah, true- I failed to remove a development test. Fixed.